File: Second Year/2nd Semester/Medii de Proiectare si Programare/C#/Laborator_1/domain/Result.cs
using System;
using System.Collections.Generic;
using System.Globalization;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Laborator_1.domain
{
    public class Result :Entity<int>
    {
        public Participant Participant { get; set; }
        public Referee Referee { get; set; }
        public int Points { get; set; }
        public string Activity { get; set; }

        public Result(Participant participant, Referee referee, int points, string activity)
        {
            Participant = participant;
            Referee = referee;
            Points = points;
            Activity = activity;
        }
    }
}

File: Second Year/2nd Semester/Medii de Proiectare si Programare/Java/Laborator_1/src/main/java/domain/Result.java
package domain;

public class Result extends Entity<Integer>{
    private Participant participant;
    private Referee referee;
    private String activity;
    private int points;

    public Result(Integer integer, Participant participant, Referee referee, String activity, int points) {
        super(integer);
        this.participant = participant;
        this.referee = referee;
        this.activity = activity;
        this.points = points;
    }

    public Participant getParticipant() {
        return participant;
    }

    public void setParticipant(Participant participant) {
        this.participant = participant;
    }

    public Referee getReferee() {
        return referee;
    }

    public void setReferee(Referee referee) {
        this.referee = referee;
    }

    public String getActivity() {
        return activity;
    }

    public void setActivity(String activity) {
        this.activity = activity;
    }

    public int getPoints() {
        return points;
    }

    public void setPoints(int points) {
        this.points = points;
    }
}
